msi/release.sh

```diff
@@ -161,7 +161,7 @@ BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
                 <File Id="GitExe" Source="cmd\\git.exe" />
             </Component>
             <Component Directory="BinFolder">
-                <File Source="mingw64\\share\\git\\compat-bash.exe" Name="sh.exe" />
+                <File Source="mingw$BITNESS\\share\\git\\compat-bash.exe" Name="sh.exe" />
```

Comment:
Please move this into a separate commit (or fix up the commit that introduced the bug, if it is part of this Pull Request).